.github/workflows/server-tests-postgres.yml

```
@@ -0,0 +1,112 @@
+name: Server (Postgres)
+
+on:
+  pull_request:
+    paths-ignore:
+      - '**.js'
+      - '**.md'
+      - '**.html'
+    types: [opened, labelled, synchronize, reopened]
```

actionlint (1.7.4)

invalid activity type "labelled" for "pull_request" Webhook event. available types are "assigned", "auto_merge_disabled", "auto_merge_enabled", "closed", "converted_to_draft", "demilestoned", "dequeued", "edited", "enqueued", "labeled", "locked", "milestoned", "opened", "ready_for_review", "reopened", "review_request_removed", "review_requested", "synchronize", "unassigned", "unlabeled", "unlocked"

(events)